internal/driver/webui.go

```diff
@@ -63,16 +63,40 @@ func serveWebInterface(port int, p *profile.Profile, o *plugin.Options) error {
 		// only allow requests from local host
 		wrap = checkLocalHost
```

Comment:
I updated checkLocalHost to be skipped if users wants to listen on a non-localhost host.